build/tools/changelog.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+console.log('Collecting Change Log');
+
+let since = '>=2019-05-03';
+
+const request = require('request-promise-native');
+const repo = 'cloudfoundry-incubator/stratos';
+const fs = require('fs');
+
+let url = 'https://api.github.com/search/issues?q=state:closed+repo:' + repo + '+updated:' + since;
+url = url + '&per_page=100';
```

Comment:
Is it possible to add a merge into param to catch just `v2-master` work?

Reply:
Yeah - could do a lot more, but just created this quickly and thought I'd commit it - we need to make separate API calls for each PR to get that info.. would be nice to ignore the ones that were closed and not merged. Also would be good to detect the 'Fixes ...' and only include the issue and not the PR that fixed it.

Finally, would be good to parse the existing changes document and exclude any that are already mentioned, just in case the date filter goes back too far.